flask/db.py

```diff
@@ -46,26 +46,21 @@ def get_products():
         with sentry_sdk.start_span(op="get_products", description="db.connect"):
             connection = db.connect()
 
-        with sentry_sdk.start_span(op="get_products", description="db.query") as span:
-            n = weighter(operator.le, 12)
-            products = connection.execute(
-                "SELECT *, pg_sleep(%s) FROM products" % (n)
-            ).fetchall()
-            span.set_tag("totalProducts",len(products))
```

Comment:
@thinkocapo just out of curiosity did logging `span = sentry_sdk.Hub.current.scope.spans` return any clues here as to what was currently on scope if anything?

I don't demo this myself so I think it is fine to move forward here.

Reply:
I didn't try logging it but I remember setting the _tag and _data would not work anymore, once we get rid of the custom span. There becomes a confusion if it's referring to the preceding span for db.connect() or is it for the auto instrumented span for sqlalchemy SELECT * FROM PRODUCTS. @dgbailey